lib/client.js

```diff
@@ -1938,6 +1938,37 @@ MatrixClient.prototype.resolveRoomAlias = function(roomAlias, callback) {
 
 /**
  * @param {string} roomId
+ * @param {module:client.callback} callback Optional.
+ * @return {module:client.Promise} Resolves: TODO
+ * @return {module:http-api.MatrixError} Rejects: with an error response.
+ */
+MatrixClient.prototype.getRoomVisibility = function(roomId, callback) {
+    var path = utils.encodeUri("/directory/list/room/$roomId", {
+        $roomId: roomId
+    });
+    return this._http.authedRequest(callback, "GET", path);
+};
+
+/**
+ * Enable or disable a push notification rule.
+ * @param {string} roomId
+ * @param {string} visibility
```

Comment:
`string`, really? I'd have thought a simple `boolean` would suffice. What's the third/fourth/etc options?